docs/contributing/database-migrations/edd.mdx

```diff
@@ -32,73 +29,76 @@ For background on this decision please see the [Evolutionary Database Design RFD
-      {" "}
-      https://martinfowler.com/articles/evodb.html{" "}
-    </a>
-  </div>
-</div>
-
-To add terminology to compliment the [above diagram](./edd#destructive-changes), migrations that are
-a part of "Deploy new changes, migrate data, put in scaffolding code" are considered _Initial_
-migrations. Migrations that are run during the Transition Phase are considered _Transition_
-migrations. And the migrations that run as a part of "Remove old schema, scaffolding code" are
-considered _Finalization_ migrations. The definitions of each are helpful when discussing the type
-of migration in relation to orchestrating them during a deployment.
-
-### Initial migrations
-
-- Compatible with _X.1.0_ **and** _X.2.0_ application code changes
-- Represents the beginning of a database change
-- Updates the database schema to support any new functionality while also maintaining old
-  functionality
-- Supports both the previous version of code and the one being upgraded to
-- Run during upgrade
-- Must execute quickly to minimize downtime.
-
-### Transition migrations
-
-- Compatible with _X.1.0_ **and** _X.2.0_ application code changes
-- The time between initial migration and finalization migration
-- Exists to provide an opportunity to rollback server to _X.1.0_ version prior to breaking changes
+Any change that cannot be done in a non-destructive manner is a destructive change. This can be as
+simple as adding a non nullable column where the value needs to be computed from existing fields, or
+renaming an existing column. To handle destructive changes it's necessary to break them up into
+three phases: _Start_, _Transition_ and _End_ as shown in the diagram below.
+
+<figure>
+
+![Refactoring Stages](./transitions.png)
+
+<figcaption>Refactoring Phases</figcaption>
+
+</figure>
+
+It's worth noting that the _Refactoring Phases_ are usually rolling, and the _End phase_ of one
+refactor is the _Transition phase_ of another. The table below details which application releases
+needs to be supported during which database phase.
+
+| Database Phase | Release X | Release X+1 | Release X+2 |
+| -------------- | --------- | ----------- | ----------- |
+| Start          | ✅        | ❌          | ❌          |
+| Transition     | ✅        | ✅          | ❌          |
+| End            | ❌        | ✅          | ✅          |
+
+### Migrations
+
+The three different migrations described in the diagram above are, _Initial migration_, _Transition
+migration_ and _ Finalization migration_ and are described in more details in this section.
+
+#### Initial migration
+
+The initial migration runs before the code deployment, and it's purpose is to add support for
+_Release X+1_ without breaking support of _Release X_. This deployment should have execute quickly
+and not contain any costly operations to ensure zero downtime.
+
+#### Transition migration
+
+The transition migration are run sometime during the transition phase, and provides an optional data
```

Comment:
⛏️ "migrations".

Reply:
I view this from the perspective of a single "change". In that scope it's singular migrations. However a deployment may contain multiple changes, and therefore multiple migrations.